### ISSUE TITLE
Feature: Monster Starting HP

### ISSUE THREAD

**Issue by gkrieg** [johreh/gloomycompanion#67]
It would be nice if the starting monster HP were shown.  That way I don't need to get out the monster stat cards at all!  Thanks.

**Comment by sanjayvsingh** [comment]
Do you have a picture of the card? I'll be fixing another card this week and can do this at the same time. The game is at a friend's house and I take pictures of the cards as we find differences - I want to test against the actual card before submitting the change. Thanks.

**Comment by nickv2002** [comment]
I think @gkrieg was asking for a bigger feature request of having the monster starting HP for all the monsters (at all scenario levels). I concur that this would be a great feature, but it's not a quick fix. There would need to be some UI to display this info.

**Comment by gkrieg** [comment]
Yes, @nickv2002 is correct.  I currently need to pull out each of the stat cards for the monsters in the scenario just so I can get their starting health stat (I use dice to track health remaining).  If that were displayed in the app, I would no longer need to pull those cards out at all, which would be convenient.

**Comment by sanjayvsingh** [comment]
Oh sorry, I had two issues open and posted that comment into the wrong one.

**Comment by Nailer** [comment]
@nickv2002 it is actually a quick fix. I did it in an hour earlier today. Forgot to check the status of this repo, and accidentally implemented it after it's already done...

**Comment by gkrieg** [comment]
Does anyone know when the changes to the repo actually show up at https://johreh.github.io/gloomycompanion/ ?

**Comment by Nailer** [comment]
@gkrieg I was wondering the same thing. I guess a manual push has to be done. In the meantime, feel free to [replace your logic.js with the content of this one.](https://github.com/johreh/gloomycompanion/files/1766231/logic.zip) (If you're for example having a Gloomhaven session today and need it immediately). My version only has changes in the .js file and is easier to swap out in a pinch.

**Comment by GinoGalotti** [comment]
Isn't this already covered by https://github.com/johreh/gloomycompanion/pull/52? I'll push it to the public page soon. But if you want something different, just speak up!

**Comment by Nailer** [comment]
@GinoGalotti it is covered! :) As you saw I accidentally implemented it after #52 was committed since it wasn't pushed to the main page.
